Makefile

```diff
@@ -340,8 +343,8 @@ PROMTOOL_VERSION ?= 2.44.0
 .PHONY: promtool
 promtool: $(PROMTOOL)
 $(PROMTOOL): $(LOCALBIN)
-	test -s $(PROMTOOL) || curl -sSfL "https://github.com/prometheus/prometheus/releases/download/v$(PROMTOOL_VERSION)/prometheus-$(PROMTOOL_VERSION).linux-amd64.tar.gz" | \
- 		tar xvzf - --directory=$(LOCALBIN) "prometheus-$(PROMTOOL_VERSION).linux-amd64"/promtool --strip-components=1
+	test -s $(PROMTOOL) || curl -sSfL "https://github.com/prometheus/prometheus/releases/download/v$(PROMTOOL_VERSION)/prometheus-$(PROMTOOL_VERSION).linux-$(ARCH).tar.gz" | \
```

Comment:
I'd suggest also to check and set OS name, e.g. it could be `darwin`.

Reply:
@codingben the changes are implemented. What do you think?

Reply:
Cool thanks :)

Reply:
Great! Thanks for the feedback :)